Calculator.Tests/Calculator.Tests/Pages/CalculatorPage.cs

```diff
@@ -14,15 +14,62 @@ public CalculatorPage(IWebDriver driver)
             this.driver = driver;
         }
 
-        public IWebElement CalculateBtn => driver.FindElement(By.XPath("//button [@id = 'calculateBtn']"));
+        private By CalculateBtnLocator = By.XPath("//button [@id = 'calculateBtn']");
+
+        public IWebElement CalculateBtn => driver.FindElement(CalculateBtnLocator);
+
+        public IWebElement SettingsBtn => driver.FindElement(By.XPath("//button[text()='Settings']"));
+
+        public SettingsPage OpenSettingsPage()
+        {
+            SettingsBtn.Click();
+            return new SettingsPage(driver);
+        }
 
         public void Calculate()
         {
             CalculateBtn.Click();
-            Thread.Sleep(1000);
+            new WebDriverWait(driver, TimeSpan.FromSeconds(10))
+            .Until(SeleniumExtras.WaitHelpers.ExpectedConditions.ElementToBeClickable(CalculateBtnLocator));
+            Thread.Sleep(200);
+        }
+        public IWebElement DepositAmountFld => driver.FindElement(By.XPath("//td[2]//input[@id = 'amount']"));
+
+        public IWebElement RateOfInterestFld => driver.FindElement(By.XPath("//input [@id = 'percent']"));
+
+        public IWebElement InvestmentTermFld => driver.FindElement(By.XPath("//input [@id = 'term']"));
+
+        public IWebElement FinancialYearRadio365 => driver.FindElement(By.XPath("//input[@type][2]"));
+
+        public IWebElement FinancialYearRadio360 => driver.FindElement(By.XPath("//input[@type][1]"));
+
+        public IWebElement Income => driver.FindElement(By.XPath("//input [@id = 'income']"));
+
+        public IWebElement InterestEarned => driver.FindElement(By.XPath("//input [@id = 'interest']"));
+
+        public IWebElement EndDate => driver.FindElement(By.XPath("//input [@id = 'endDate']"));
+
+        public IWebElement InterestEarnedLayout => driver.FindElement(By.XPath("//th [contains(text(), 'Interest earned')]"));
+
+        public IWebElement ActualIncome => driver.FindElement(By.XPath("//input [@id = 'income']"));
```

Comment:
It's better to change this property type to 'string' and return the text value. This works if you don't need the control itself, only its text.
```suggestion
        public string ActualIncome => driver.FindElement(By.XPath("//input [@id = 'income']")).GetAttribute("value");
```

Reply:
done, thanks